solutions/sort_numbers.py

```diff
@@ -0,0 +1,30 @@
+def sort_numbers(numbers):
```

Comment:
You forgot to add input/output annotations

Reply:
Thank you! Added now!